api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt

```diff
@@ -0,0 +1,6 @@
+package wu.seal.jsontokotlinclass.server.models.routes.addhit
+
+import wu.seal.jsontokotlinclass.server.models.routes.base.BaseResponse
+
+class AddHitResponse(data: Void?, error: Boolean, errorCode: Int, message: String) : BaseResponse<Void>(data, error, errorCode, message) {
```

Comment:
Data Type is `Void`? Hey, what's the meaning?

Reply:
As adding a hit is simply an insertion event, the `data` in the response `JSON`  from the `addHit` route will be null.

Reply:
Sure! 😄